templates/error.html

```diff
@@ -0,0 +1,139 @@
+{% extends 'base.htm' %}
+
+{% block content %}
+    <!-- This template was made by Colorlib (https://colorlib.com)
+         Please visit our website for more awesome templates, themes and tools. -->
+	<div id="notfound">
+		<div class="notfound">
+			<div class="notfound-404">
+				<h1>Oops!</h1>
+				<h2>
+                    {% block message %}{% endblock %}
+                </h2>
+			</div>
+			<a href="/">Go TO Homepage</a>
```

Comment:
Let's remove this link.  The logo in the navbar already functions as a link to the homepage, and if the user wants to go back to the previous page (before the error happened), they can just use the browser's back button.